### DISCUSSION TITLE
Can't submit sponsor profile for approval

### DISCUSSION THREAD

**Discussion by vagabondrev** [community/community#27546] [Other Feature Feedback, Questions, & Ideas]
I've completed my profile for sponsors, but it shows an error that is preventing me from submitting for approval.

"Please check your name, date of birth, and address in your Stripe Connect account for any errors."

I've checked my Stripe account multiple times but cannot find any errors; how do I get past this?

**Comment by jfullstackdev** [comment]
you can ask those who completed it here, because they said, it's a different Stripe account   

https://github.com/orgs/community/discussions/8712

https://github.com/orgs/community/discussions/27414

**Reply by vagabondrev** [reply]
I set up my Stripe account using the link provided as they suggested, that's when I get this error. 

So I tried creating a separate account with Stripe, but I could find no way to connect that account with GitHub Sponsors. So I went through the steps to add my Strip account again -- thinking I could point it to my newly created account. 

It added GitHub as a new account on my Stripe dashboard, but I still have the same error preventing me from submitting my application.

What am I doing wrong?

![Screen Shot 2022-07-28 at 5 55 00 PM](https://user-images.githubusercontent.com/15913124/181794480-67a248ce-294e-4231-b51e-5c3b476dd3f6.png)
.
![Screen Shot 2022-07-29 at 10 37 26 AM](https://user-images.githubusercontent.com/15913124/181794783-8a749acc-8e7c-43e3-8a2f-ed3e20a1ce11.png)
![Screen Shot 2022-07-29 at 10 37 43 AM](https://user-images.githubusercontent.com/15913124/181794797-42beeffa-44fa-4778-bdd0-3f0555f35fbe.png)

**Reply by jfullstackdev** [reply]
as mentioned at those threads, even if the country is not fully supported by Stripe, GitHub will handle all the rest, so it's an account from GitHub but I'm not very sure if that should be connected to your existing Stripe account, let's see, hopefully someone knows the answer and will see this post

but I'll be linking this to the two threads, so they will see this

**Reply by Logvin** [reply]
[vagabondrev](https://github.com/vagabondrev) I am having the *exact* same issue. I never had a Strip account. I clicked the link in the Github Sponsors page, created my account, verified it, can log in fine - but I can't get past the same step you are stuck with. I'm in the USA. I'd upload screenshots but mine would be identical to yours. Hope we find some help soon!

**Reply by saman1368** [reply]
@vagabondrev @Logvin

**Reply by EduardoP103** [reply]
watch this
https://github.com/orgs/community/discussions/8712

**Reply by Logvin** [reply]
Yeah I'm watching it, but its very different. This person already had a Stripe account, and they made it past the step that vagabondrev and I are stuck at. Plus that post is about PH, not USA. I don't believe these issues are related.

**Reply by BrianPugh** [reply]
I'm having the exact same issue.

**Reply by Logvin** [reply]
@BrianPugh @vagabondrev:

Update! I received a response just now from Github support: they looked at my github sponsor profile and said it was fully ready to go and I should look at my dashboard. I went to my dashboard, and what do you know... the message previously there was gone and I was ready to submit my profile to them.

If you are wondering what happened, there is a simple word to describe it: Gaslighting. They hit some button to manually advance me and then acted like I am a fool for not seeing the next step. Rather rude and annoying IMO, but I guess the goal was to fix the issue and it is now fixed. If they would simply put a label that says "Stripe verification can take 2-3 business days" I would have been perfectly fine...

**Reply by vagabondrev** [reply]
@Logvin it looks like they did the same for me. I had to update a setting on my Stripe account, but now I was able to submit for approval. Thanks to all for the help getting this the attention we needed.

**Comment by inosgun** [comment]
Apa yg dimaksut no router

**Comment by queenofcorgis1** [comment]
Hi there! 

Since this concerns sensitive financial information, we consider these to be personal account questions that should be routed through our [Support team](https://support.github.com/contact) so we can answer them privately.

**Comment by alex-titarenko** [comment]
Any resolution here? I have the exact same error. I did not have Stripe account before and created one from the link in GitHub.

**Reply by Logvin** [reply]
Not yet; I did ask their Support Team, but it has only been 13 hours since I did so. I'll do my best to respond to this post when if I get more info / a resolution!

**Reply by Logvin** [reply]
Update! I received a response just now from Github support: they looked at my github sponsor profile and said it was fully ready to go and I should look at my dashboard. I went to my dashboard, and what do you know... the message previously there was gone and I was ready to submit my profile to them.

If you are wondering what happened, there is a simple word to describe it: Gaslighting. They hit some button to manually advance me and then acted like I am a fool for not seeing the next step. Rather rude and annoying IMO, but I guess the goal was to fix the issue and it is now fixed. If they would simply put a label that says "Stripe verification can take 2-3 business days" I would have been perfectly fine...

**Comment by Logvin** [comment]
Update! I received a response just now from Github support: they looked at my github sponsor profile and said it was fully ready to go and I should look at my dashboard. I went to my dashboard, and what do you know... the message previously there was gone and I was ready to submit my profile to them.

If you are wondering what happened, there is a simple word to describe it: Gaslighting. They hit some button to manually advance me and then acted like I am a fool for not seeing the next step. Rather rude and annoying IMO, but I guess the goal was to fix the issue and it is now fixed. If they would simply put a label that says "Stripe verification can take 2-3 business days" I would have been perfectly fine...
